app/src/main/java/io/gnosis/safe/ui/StartActivity.kt

```diff
@@ -70,6 +73,9 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler {
 
     private fun setSafe(safe: Safe) {
         with(toolbarBinding) {
+            safeImage.setOnClickListener {
+                Navigation.findNavController(this@StartActivity, R.id.nav_host).navigate(ShareSafeDialogDirections.actionGlobalShareSafeDialog())
```

Comment:
It is possible to navigate using dialog id just like we do it with safeSelectionDialog:
```Navigation.findNavController(this@StartActivity, R.id.nav_host).navigate(R.id.shareSafeDialog)```
I think it is more readable this way. And we won't need another global action

Reply:
I was just using the `safeArgs`, but I agree in this case the `id` is shorter and looks nicer.